apps/web/src/components/(marketing)/open-build-template-bento.tsx

```diff
@@ -13,6 +13,22 @@ import cardTemplateFigure from '~/assets/card-template-figure.png';
 export type OpenBuildTemplateBentoProps = HTMLAttributes<HTMLDivElement>;
 
 export const OpenBuildTemplateBento = ({ className, ...props }: OpenBuildTemplateBentoProps) => {
+  const cardData = [
```

Comment:
Same thing, better as JSX since it's more natural to edit the content 😄

Reply:
Done